jest/setupFile.js

```diff
@@ -30,3 +32,5 @@ jest.mock(
   () => "push-notification-ios",
 )
 jest.mock("rn-fetch-blob", () => "Blob")
+
+initializei18next(["en"])
```

Comment:
This needs to happen for all of our test matches to work, also the `RTLEnabledText` accesses the direction of the text, this gets initialized with `i18Next`